apps/coursition/src/app/actions.ts

```diff
@@ -45,8 +49,91 @@ export const createUser = async ({ email, password }: { email: string; password:
     })
 }
 
-export const getInTouch = async (formdata: FormData) => {
-  const data = Object.fromEntries(formdata)
-  console.log({ data })
-  // todo: call resend helper
+export const getInTouch = async (prevState: { message: string }, formdata: FormData) => {
+  const data = Object.fromEntries(formdata) as Record<string, string>
+
+  const { email = '', firstName, lastName, comment } = data
+  return await send({
+    react: Components.GetInTouchEmailTemplate({ email, fullName: `${firstName} ${lastName}`, comment }),
+    from: 'Acme <onboarding@resend.dev>',
+    to: [''], // company's email
+    subject: 'Get In Touch',
+  })
+    .then(() => ({ message: 'Your invitation has been sent' }))
+    .catch(() => ({ message: "Sorry, your invitation wasn't sent" }))
+}
+
+export const sendResetPassword = async (
+  _: { step: number; error?: string | null },
+  formdata: FormData,
+): Promise<{ step: number; error?: string }> => {
+  try {
+    const email = formdata.get('email') as string
+
+    const user = await prisma.user.findFirst({ where: { email } })
+
+    if (!user?.id) return { error: 'user not found', step: 1 }
+
+    const record = (await prisma.passwordReset
+      .create({
+        data: {
+          id: `re_${nanoid(6)}`,
+          /** 4 Hours */
+          expires_at: new Date(Date.now() + 1000 * 60 * 60 * 4),
+          is_used: false,
+          secret: `sec_${nanoid(12)}`,
+          userId: user.id,
+        },
+      })
+      .catch((err) => {
+        console.log({ err })
+        return { step: 1, error: JSON.stringify(err) }
+      })) as PasswordReset
+
+    if (!record?.id) return { error: 'error generating reset link', step: 1 }
+
+    return await send({
+      from: 'Acme <onboarding@resend.dev>', // todo: change
+      to: [email],
+      subject: 'Reset your password',
```

Comment:
**blocker** We should add prefix like "Coursition - Reset your password" or something like that.